.github/workflows/build_db.yaml

```diff
@@ -0,0 +1,75 @@
+name: Create & Update DB
+
+on: [pull_request, workflow_dispatch]
+
+jobs:
+  build-and-push:
+    runs-on: ubuntu-latest
+    defaults:
+        run:
+          shell: bash -el {0}
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
+
+    - name: Build environment
+      uses: ./.github/actions/build-conda-env
+      with:
+        env_file: environment.yaml
+
+    - name: Activate environment & check conda info
+      run: |
+        conda activate $(grep name environment.yaml | awk '{ print $2}')
+        conda info
+        echo "Install cfdb"
+        pip install -e .
+
+    - name: Retrieve previous DB table from artifacts
+      uses: actions/download-artifact@v2
+      with:
+        name: cfdb
+        path: /home/runner/work/czi-conda-forge-db/cf-database.db
+
+    - name: Create runtest directory
+      if: always()
```

Comment:
Why do you need all these `always()` calls?

Reply:
Because of the order of which everything is done right now, to use the artifacts as a form of caching the previous state of the DB in the workflow above, we need first to download it from the artifacts store. The issue with the very first run is that such object does not exists, and the artifact action above will return an error and fail when  that happens, which thus hinder the progress of the pipeline that would have created the file and pushed it